specs/Deployment/DeploymentAPI.md

```diff
@@ -143,18 +161,16 @@ by using DeploymentInitializeOptions object and setting ForceDeployment option b
 this API. This option when set will shut down the application(s) associated with WinAppSDK packages,
 update the WinAppSDK packages and restart the application(s).
```

Comment:
If I don't set ForceDeployment true, the deploy will fail? And then the app should interact with the to say "on next reboot" or some such?